client/my-sites/customer-home/cards/launchpad/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { CircularProgressBar } from '@automattic/components';
+import { useLaunchpad } from '@automattic/data-stores';
+import { Launchpad, Task } from '@automattic/launchpad';
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { getSiteSlug } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+
+import './style.scss';
+
+interface CustomerHomeLaunchpadProps {
+	siteSlug: string | null;
+	checklistSlug: string;
+	taskFilter: ( tasks: Task[] ) => Task[];
+}
+
+const CustomerHomeLaunchpad = ( {
+	siteSlug,
+	checklistSlug,
+	taskFilter,
+}: CustomerHomeLaunchpadProps ): JSX.Element => {
+	const translate = useTranslate();
+	const {
+		data: { checklist },
+	} = useLaunchpad( siteSlug, checklistSlug );
```

Comment:
As for the `LaunchpadBlogFlow` component, can we rely on `useSelector()` instead of `connect()`?
```suggestion
const CustomerHomeLaunchpad = ( {
	checklistSlug,
	taskFilter,
}: CustomerHomeLaunchpadProps ): JSX.Element => {
	const translate = useTranslate();
	const siteId = useSelector( getSelectedSite );
	const siteSlug = useSelector( ( state: AppState ) => getSiteSlug( state, siteId ) );
	const {
		data: { checklist },
	} = useLaunchpad( siteSlug, checklistSlug );
```